@here/harp-omv-datasource/lib/OmvDataSource.ts

```diff
@@ -276,20 +276,23 @@ function completeDataSourceParameters(
 
 export class OmvDataSource extends TileDataSource<OmvTile> {
     private readonly m_decoderOptions: OmvDecoderOptions;
+    private m_params: OmvWithRestClientParams | OmvWithCustomDataProvider;
 
-    constructor(private m_params: OmvWithRestClientParams | OmvWithCustomDataProvider) {
-        super(m_params.tileFactory || new TileFactory(OmvTile), {
-            styleSetName: m_params.styleSetName ?? "omv",
```

Comment:
this would set the incorrect `styleSetName`, because the merging of the defaults happens later